src/ForgotPasswordController.js

```diff
@@ -134,16 +134,19 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, ContactSupport,
             this.$el.addClass('forgot-password-sms-enabled');
             formChildren.push(this.createRecoveryFactorButton('sms-button', 'password.forgot.sendText',
               Enums.RECOVERY_FACTOR_TYPE_SMS, this));
+            this.setDefaultFactorType(Enums.RECOVERY_FACTOR_TYPE_SMS);
           }
           if (callEnabled) {
             this.$el.addClass('forgot-password-call-enabled');
             formChildren.push(this.createRecoveryFactorButton('call-button', 'password.forgot.call',
               Enums.RECOVERY_FACTOR_TYPE_CALL, this));
+            this.setDefaultFactorType(Enums.RECOVERY_FACTOR_TYPE_CALL);
           }
           if (emailEnabled) {
             this.$el.addClass('forgot-password-email-enabled');
             formChildren.push(this.createRecoveryFactorButton('email-button', 'password.forgot.sendEmail',
               Enums.RECOVERY_FACTOR_TYPE_EMAIL, this));
+            this.setDefaultFactorType(Enums.RECOVERY_FACTOR_TYPE_EMAIL);
```

Comment:
Double checking - is this the right order of preference (SMS, then call, then email)? 

Edit: Looking at the test cases, this is indeed the intended behavior. Good!